gctree/branching_processes.py

```diff
@@ -987,3 +964,230 @@ def f(x):
     if not result.success:
         warnings.warn("optimization not sucessful, " + result.message, RuntimeWarning)
     return result.x[0], result.x[1]
+
+
+def lltree(cm_counts, p: np.float64, q: np.float64) -> Tuple[np.float64, np.ndarray]:
```

Comment:
Does this need to be a public function in the API that is shown in docs? If not, suggest renaming `_lltree`.